Packer.php

```diff
@@ -114,7 +114,7 @@ public function setBoxesLimited($limited = true) {
   }
 
   /**
-   * Boolean If building nests or not
+   * Boolean If boxes are limited or not | It also determines if packer is doing nesting or not
```

Comment:
Do you need to refer to nests here?  Does this library even know what a nest is?